shared/elements/gaia_buttons/style.css

```diff
@@ -8,17 +8,25 @@ gaia-buttons {
   display: flex;
   justify-content: center;
   align-items: center;
+  height: 7rem;
+  box-sizing: border-box;
+  pointer-events: auto;
+}
+
+gaia-buttons:not(.inline) {
   position: absolute;
   left: 0;
   right: 0;
   bottom: 0;
   z-index: 10;
-  height: 7rem;
-  box-sizing: border-box;
-  padding: 0 1.5rem 0 0;
   border-top: 0.1rem solid #e7e7e7;
+  padding: 0 1.5rem 0 0;
   background-color: #f4f4f4;
-  pointer-events: auto;
+}
+
+gaia-buttons.inline a,
+gaia-buttons.inline button {
+  margin: 0;
```

Comment:
We already add it to the container, so I don't think we need any special type of button. (I don't want to have to apply the class to the container and the child node as well).

The buttons by default have some styling applied to compensate for the default container padding. We could probably clean this up, but it might be a larger refactoring. Let me know exactly what you want me to change (if anything), and I can do so.